app/views/users/index.html.erb

```diff
@@ -1,13 +1,24 @@
-<div class="each-user-div">
-  <img src="<%= f.photo %> " alt = "<%= f.name %>"  />
-
-     <div>
-        <p>   UserName: <%= f.name %></p>
-         <p>  Number of Posts: <%= f.postscounter %></p>
-         <button>
-         <%= link_to("See Profile", user_path(f)) %>
-         </button>
-         <%= link_to "Change Password", edit_user_registration_path %>
-         
-    </div>
 
+    <% @users.each do |f| %>
+      <div class="each-user-div">
+      <img src="<%= f.photo %> "  />
+    
+         <div>
+            <p>   UserName: <%= f.name %></p>
+             <p>  Number of Posts: <%= f.postscounter %></p>
+             <button>
+             <%= link_to "See Profile", user_path(f) %>
+             </button>
+        </div>
+       
+      </div>
+      <%= link_to "Change Password", edit_user_registration_path %>
+    
+    
+    <% end %>
```

Comment:
- [ ] [OPTIONAL] I wasn't able to. sign up a user until I add `password` and `password_confirmation` [`devise_parameter_sanitizer.permit(:sign_up)`](https://github.com/MICHAELMUNAVU83/blog/blob/2d40e086e8a31e845b03dbe55e6c6f68be62e47d/app/controllers/application_controller.rb#L9)
please make sure to fix that